packages/dd-trace/src/appsec/rule_manager.js

```diff
@@ -10,11 +11,16 @@ let appliedRulesetId
 let appliedRulesOverride = new Map()
 let appliedExclusions = new Map()
 let appliedCustomRules = new Map()
+let appliedActions = new Map()
 
 function applyRules (rules, config) {
   defaultRules = rules
 
   waf.init(rules, config)
+
+  if (rules.actions) {
+    blocking.updateBlockingConfiguration(rules.actions.filter(action => action.id === 'block'))
```

Comment:
You could get only first match

```suggestion
    blocking.updateBlockingConfiguration(rules.actions.find(action => action.id === 'block'))
```

Reply:
If a blocking configuration has been set before, no receiving here a `block` action will be considered as a reset to the default config?
Or, once a user has configured a blocking action, it is always sent?

Reply:
This method is called only on enable appsec. 
`actions` flag is in rules, only when the customer is defining a custom rules files, in that case, the rules RC is not enabled.